### PR TITLE
RUNNER-7075: Replace WSK usage by AIO CLI

### PR DESCRIPTION
**Pull request by joni-jones**
## Description

 - Replaced wsk with aio CLI

### CODE REVIEW

**Review by selfxp — APPROVED** [review]
💯

**Review by wllbo — APPROVED** [review]

File: src/pages/guides/using/creating_actions.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ For your code to execute as an *action* on Adobe I/O Runtime, your code has to c
 - You either call your function *main* or you export the entry point as *main*. This is the function that will be executed when it is invoked.
 - Your function accepts valid JSON objects as input and produces valid JSON objects as output, if needed
 
-You have to configure two tools on your machine to create and invoke actions from your machine: **wsk CLI** and **wskdeploy CLI.** Refer to the [wsk CLI](../tools/wsk_install.md) page for how to install and configure it. 
+You have to configure two tools on your machine to create and invoke actions from your machine: **aio CLI** and **wskdeploy CLI.** Refer to the [aio CLI](../tools/cli_install.md) page for how to install and configure it.
```
Comment (buskamuza) [inline src/pages/guides/using/creating_actions.md:7]: Maybe we can move `wskdeploy CLI` to the section where it's used?
1. Here we mention it, but don't provide a link to the docs how to install it
2. it's not used for single action deployment, so it's not needed for all use cases (but we say `You have to` here).

It's another discussion whether we still need to reference this tool or provide steps for useing `aio` CLI for deploying complex actions.

**Review by buskamuza — APPROVED** [review]
The change is great 🥳 , so feel free to merge.
Please check my comments to see if you'd like to address some of them now. And can we create tickets for others (if they make sense)?

File: src/pages/guides/using/creating_actions.md (hunk 1 above)
Comment (joni-jones) [reply src/pages/guides/using/creating_actions.md:7]: `wskdeploy` is used in the same file, I'm not sure what is the case for it, so decided to don't touch it.

File: src/pages/guides/tools/index.md [hunk 2]
```diff
@@ -2,6 +2,6 @@
 
 The most important tools you&rsquo;ll need when working with Adobe I/O Runtime are:
 
-- [aio CLI](cli_install.md): helps you manage your namespaces and the authentication for the `wsk` CLI
+- [aio CLI](cli_install.md): helps you manage your namespaces and the authentication for the `aio` CLI
 - [wsk CLI](wsk_install.md): the main interface for managing your actions/packages/rules/triggers and get access to activation results/errors/logs
```
[the remark this answers is not in the document]
Comment (joni-jones) [reply src/pages/guides/tools/index.md:6]: It's a good question, but I think out of the scope of this PR.

File: src/pages/guides/tools/index.md [hunk 3]
```diff
@@ -2,6 +2,6 @@
 
 The most important tools you&rsquo;ll need when working with Adobe I/O Runtime are:
 
-- [aio CLI](cli_install.md): helps you manage your namespaces and the authentication for the `wsk` CLI
+- [aio CLI](cli_install.md): helps you manage your namespaces and the authentication for the `aio` CLI
```
[the remark this answers is not in the document]
Comment (joni-jones) [reply src/pages/guides/tools/index.md:5]: I'm not sure as well, probably somebody didn't come up with a good description.

> not sure AppBuilder users need to know what .wskprops is.

Developers still can manage their actions without having an App Builder app and they will need to modify `.wskprops`. For instance, to manage single actions in different NS I never use App Builder, it's an overkill.

File: src/pages/guides/using/creating_actions.md (hunk 1 above)
Comment (buskamuza) [reply src/pages/guides/using/creating_actions.md:7]: I mean, it's used in one of the sections ("Deploying from ZIP archive" or so"). I thought maybe we can reference it there. But not critical.

Comment (joni-jones) [reply src/pages/guides/using/creating_actions.md]: I changed it.

**Pull request merged**